### PR TITLE
main and improved circuit methods

### PR DESCRIPTION
**Pull request by tmjo**
Merged into my master which is in synch with upstream master, then new PR.

### CODE REVIEW

File: easee/charger.py [hunk 1]
```diff
@@ -136,3 +139,17 @@ async def reboot(self):
...
+
+    async def set_dynamic_current(self, currentP1: int, currentP2: int = None, currentP3: int = None):
+        """ Wrapper function to set circuit dynamic current for charger """
+        if self.circuit is not None:
+            return await self.circuit.set_dynamic_current(currentP1, currentP2, currentP3)
+        else:
+            _LOGGER.info("Circuit info must be initialized for dynamic current to be set")
+
+    async def set_max_current(self, currentP1: int, currentP2: int = None, currentP3: int = None):
+        """ Wrapper function to set circuit max current for charger """
```
Comment (fondberg) [inline easee/charger.py:151]: Maybe remove the text wrapper function to adhere to the rest if the docs?

File: easee/charger.py [hunk 2]
```diff
@@ -136,3 +139,17 @@ async def reboot(self):
     async def update_firmware(self):
         """Update charger firmware"""
         return await self.easee.post(f"/api/chargers/{self.id}/commands/update_firmware")
+
+    async def set_dynamic_current(self, currentP1: int, currentP2: int = None, currentP3: int = None):
+        """ Wrapper function to set circuit dynamic current for charger """
```
Comment (fondberg) [inline easee/charger.py:144]: Same comment as below

Comment (tmjo) [reply easee/charger.py]: Yep, done!

**Comment by fondberg** [comment]
Will release tonight.

**Pull request merged**